examples/cms/datasources/articles.json

```diff
@@ -0,0 +1,49 @@
+{
+    "id": "articles",
+    "fields": {
+        "Title": {
+            "type": "string"
+        },
+        "Content": {
+            "type": "formattedText"
+        },
+        "Reading time": {
+            "type": "number"
+        },
+        "Featured": {
+            "type": "boolean"
+        }
```

Comment:
Nit. Might be fine, but not very realistic that an API uses the same types as us. Maybe we should name the content type here `"html"` just to make that a little more clear

Reply:
We can! This was the first iteration, but it would add complexity mapping those values to ours.

I think @tom-james-watson @huntercaron may have stronger opinions

Reply:
For the sake of the example, I added it and a comment on "custom type"